forge/forge/python_codegen.py

```diff
@@ -953,7 +1000,7 @@ def write_param_parser(
         else:
             assert False, "TODO: Add other framework param parsers"
 
-    def write_pytest_function(self, module_name, input_shapes):
+    def write_pytest_function(self, module_name, framework, input_shapes, input_dtypes, activation_param_names, need_process_framework_parameters_func=False, param_file_name=None, names_params_file_name=None, named_buffers_file_name=None):
```

Comment:
If we have a long list of arguments, let's see to create a common object that we can use.